src/core/zowe/core_for_zowe_sdk/logger.py

```diff
@@ -0,0 +1,21 @@
+import logging
+import os
+
+class Log:
+    """root logger setup and a funtion to customize logger level"""
+
+    dirname = os.path.join(os.path.expanduser("~"), ".zowe/logs")
+
+    if not os.path.isdir(dirname):
+        os.makedirs(dirname)
+
+    logging.basicConfig(
+        filename=os.path.join(dirname, "python_sdk_logs.log"),
+        level=logging.INFO,
+        format="[%(asctime)s] [%(levelname)s] [%(name)s] - %(message)s",
+        datefmt="%m/%d/%Y %I:%M:%S %p",
+    )
+
+    @staticmethod
+    def setLoggerLevel(level: int):
+        logging.root.setLevel(level)
```

Comment:
I don't know if setting the level of `logging.root` is best practice here. If I understand correctly this will affect the level of all loggers, including ones that other Python packages besides the Zowe SDK may be using.

Since we create multiple loggers in each module that each have their own name, if we want to set logger level across all of them without affecting the root logger, we may need to track them in this class. Something like the following:
```suggestion
    loggers = []
    @staticmethod
    def registerLogger(name: string):
        Log.loggers.append(logging.getLogger(name))

    @staticmethod
    def setLoggerLevel(level: int):
        for logger in Log.loggers:
            logger.setLevel(level)
```